test/integration/verify_up4.go

```diff
@@ -262,6 +263,27 @@ func buildExpectedTunnelPeersEntry(client *p4rtc.Client, testdata *pfcpSessionDa
 	}, client.NewTableActionDirect(ActLoadTunnelParam, [][]byte{srcAddr, dstAddr, srcPort}), nil)
 }
 
+func buildExpectedSliceTcMeter(expectedValues sliceMeter) *p4_v1.MeterEntry {
+	meterIndex := getSliceTcMeterIndex(expectedValues.sliceID, expectedValues.TC)
+
+	meterConfig := &p4_v1.MeterConfig{
+		Cir:    int64(0),
+		Cburst: int64(0),
+		Pir:    expectedValues.rate,
+		Pburst: expectedValues.burst,
+	}
+
+	return &p4_v1.MeterEntry{
+		MeterId: 336833095,
+		Index:   &p4_v1.Index{Index: meterIndex},
+		Config:  meterConfig,
+	}
+}
+
+func getSliceTcMeterIndex(sliceID uint8, TC uint8) int64 {
```

Comment:
What about moving this function to `pfcpiface/utils.go` and re-using it among UP4 plugin and integration tests?

Reply:
We would just need a unit test to verify the proper behavior